src/jwt.ts

```diff
@@ -0,0 +1,36 @@
+interface Token {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  header: any
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  payload: any
```

Comment:
Is it possible to be more specific than `any`?

Reply:
@dennerkrans I guess we can be a little bit more specific. But since jwt tokens can include any custom claims in header and payload we cannot be too specific. Maybe this would be a slight improvement?

```
interface TokenPart {
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  [key: string]: any
}

interface Token {
  header: TokenPart
  payload: TokenPart
  signature: string
}
```

Reply:
I think we can keep it as `any` in that case.